### PR TITLE
final-projectPullRequest

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About this project

This project is about a mobile web app which displays Information about world’s continents, and its population number. 
Also, the mobile web app displays information about each country’s Information like capital, area, region, population number, language and currency. 

Besides, this mobile web app has a search navigation bar to filter country. 

🌱 For it:    I created several branches to respect the Gitflow. 

🌱 Inside the Folder src : 
    I have placed folders named components, redux, images, and imagesContinents 
    
🌱 Inside our components folder, we created our folders named: 
       - countries  - details - navbar
       
🌱 Inside our redux folder, we placed 2 folders named:   
     - actions 
     - reducers
     
🌱 We installed react components, and props. 

🌱 We installed react router, and redux.  

🌱 We installed react-bootstrap, react-icons, and bootstrap.  

🌱 We installed axios Library and imported in some of our files.  

🌱 We used react testing library and jest.  

🌱 We created files for testing and snapshots.  

🌱 We have used render site for the deployment of our project.  

🌱 We have recorded our video using loom. 

 🌱 We used linters like ESLint and Stylelint and passed linters.
 
### Goal:

The goal of the project is to understand the importance of the usage of React and its components, and props, React Router, and Redux in the creation of our mobile web apps. 
Also the importance of using external libraries like axios, and its relevance in getting data from the external API, as well as functions like actions creators asyncronous.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ bash
 
 - [ ] *New content*
 - [ ] *Testimonial section*
+- [ ] *likes icon to check which continent is most visited*
```
Comment (ademibowale) [inline README.md:150]: - [ ] You did a great job on this milestone but there are some things you still need to address 
Kindly The video contains a highlight about a piece of the code the student finds interesting or is proud of making please make a clear explanation about a pieces of the code you find interesting or proud of explanation with a cogest point of the lines of code .

Comment (ademibowale) [inline README.md:150]: - [ ] great job on this milestone please The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).please give appropriate credit to the author .

**Review by ademibowale — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @IvonneBenitesRodriguez     😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

just a little bit

## Highlights
  ✅ Nice user interface
  ✅Correct use of Github Flow
  ✅ No linter errors
✅ Descriptive Pull Request
  

## Required Changes ♻️




_Check the comments under the review._




## Optional suggestions 
   
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply README.md:150]: Good evening dear @ademibowale,
I hope this message finds you well,
can you please confirm me if I do need to make the video again showing the pieces of code 
i find interesting ?
Thank you in advance,

Comment (ademibowale) [reply README.md:150]: @IvonneBenitesRodriguez you did well on your presentation but you only need to make a bit more  explanation of your piece of the code you love ,so just make new video ,this presentation video will be useful for you later please give proper explanation of the pieces of the code you love .

Comment (IvonneBenitesRodriguez) [reply README.md:150]: Dear @ademibowale 
thank you so much for your kind reply and explanation,
I modified the acknowledgements section as you suggested,
have a beautiful week,
Best wishes

Comment (IvonneBenitesRodriguez) [reply README.md:150]: Dear @ademibowale,
Thank you so much for replying, I truly appreciate it, 
I have made a new video with the suggestion you mentioned, 
thank you again, 
Best wishes,

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ivonne,  

**React Capstone** project is complete! 🏁 Great job! Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on introducing improvements to the ReadMe!
🎲 Awesome presentation!
🎲 Nice job acknowledging [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) 
🎲 ReadMe looks great!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ademibowale) [reply README.md:150]: @IvonneBenitesRodriguez you are welcome madam

**Pull request merged**